File: Practica_11/Python/Fila_A/Artista.py
class Artista:
    def __init__(self, nombrex, cix, añosExperienciax):
        self.nombre = nombrex
        self.ci = cix
        self.añosExperiencia = añosExperienciax

    def mostrar(self):
        print("ARTISTA")
        print("Nombre:", self.nombre)
        print("C.I.:", self.ci)
        print("Años Experiencia:", self.añosExperiencia)

    def getNombre(self):
        return self.nombre

    def setNombre(self, nombre):
        self.nombre = nombre

    def getCi(self):
        return self.ci

    def setCi(self, ci):
        self.ci = ci

    def getAñosExperiencia(self):
        return self.añosExperiencia

    def setAñosExperiencia(self, añosExperiencia):
        self.añosExperiencia = añosExperiencia

File: Practica_11/src/Java/Fila_A/Artista.java
package Java.Fila_A;

public class Artista {
	String nombre;
	int ci;
	int añosExperiencia;
	
	public Artista(String nombrex, int cix, int añosExperienciax) {
		this.nombre=nombrex;
		this.ci=cix;
		this.añosExperiencia=añosExperienciax;
	}
	public void mostrar() {
		System.out.println("ARTISTA");
		System.out.println("Nombre: "+this.nombre);
		System.out.println("C.I.: "+this.ci);
		System.out.println("Años Experiencia: "+this.añosExperiencia);
	}
	public String getNombre() {
		return nombre;
	}

	public void setNombre(String nombre) {
		this.nombre = nombre;
	}

	public int getCi() {
		return ci;
	}

	public void setCi(int ci) {
		this.ci = ci;
	}

	public int getAñosExperiencia() {
		return añosExperiencia;
	}

	public void setAñosExperiencia(int añosExperiencia) {
		this.añosExperiencia = añosExperiencia;
	}
}
